### ISSUE TITLE
Ready-to-use environment YAML file configured for training on GPU or CPU.

### ISSUE THREAD

**Issue by iamshreeji-copy1** [NVIDIA/tacotron2#642]
`name: tacotron3.6_env
channels:
  - defaults
  - https://repo.anaconda.com/pkgs/main
  - https://repo.anaconda.com/pkgs/r
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - ca-certificates=2025.2.25=h06a4308_0
  - certifi=2022.12.7=py37h06a4308_0
  - ld_impl_linux-64=2.40=h12ee557_0
  - libffi=3.4.4=h6a678d5_1
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libstdcxx-ng=11.2.0=h1234567_1
  - libxcb=1.17.0=h9b100fa_0
  - ncurses=6.4=h6a678d5_0
  - openssl=1.1.1w=h7f8727e_0
  - pthread-stubs=0.3=h0ce48e5_1
  - python=3.7.16=h7a1cb2a_0
  - readline=8.2=h5eee18b_0
  - setuptools=65.6.3=py37h06a4308_0
  - sqlite=3.45.3=h5eee18b_0
  - tk=8.6.14=h993c535_1
  - wheel=0.38.4=py37h06a4308_0
  - xorg-libx11=1.8.12=h9b100fa_1
  - xorg-libxau=1.0.12=h9b100fa_0
  - xorg-libxdmcp=1.1.5=h9b100fa_0
  - xorg-xorgproto=2024.1=h5eee18b_1
  - xz=5.6.4=h5eee18b_1
  - zlib=1.2.13=h5eee18b_1
  - pip:
      - absl-py==2.1.0
      - apex==0.1
      - astor==0.8.1
      - audioread==3.0.1
      - cffi==1.15.1
      - charset-normalizer==3.4.2
      - cxxfilt==0.3.0
      - cycler==0.11.0
      - decorator==5.1.1
      - exceptiongroup==1.3.0
      - fonttools==4.38.0
      - gast==0.2.2
      - google-pasta==0.2.0
      - grpcio==1.62.3
      - h5py==3.8.0
      - idna==3.10
      - importlib-metadata==6.7.0
      - importlib-resources==5.12.0
      - inflect==0.2.5
      - iniconfig==2.0.0
      - joblib==1.3.2
      - keras-applications==1.0.8
      - keras-preprocessing==1.1.2
      - kiwisolver==1.4.5
      - lazy-loader==0.4
      - librosa==0.9.1
      - llvmlite==0.39.1
      - markdown==3.4.4
      - markupsafe==2.1.5
      - matplotlib==3.5.3
      - msgpack==1.0.5
      - numba==0.56.4
      - numpy==1.21.6
      - nvidia-cublas-cu11==11.10.3.66
      - nvidia-cuda-nvrtc-cu11==11.7.99
      - nvidia-cuda-runtime-cu11==11.7.99
      - nvidia-cudnn-cu11==8.5.0.96
      - opt-einsum==3.3.0
      - packaging==24.0
      - pillow==9.5.0
      - pip==21.0
      - platformdirs==4.0.0
      - pluggy==1.2.0
      - pooch==1.8.2
      - protobuf==3.20.1
      - pycparser==2.21
      - pyparsing==3.1.4
      - pytest==7.4.4
      - python-dateutil==2.9.0.post0
      - requests==2.31.0
      - resampy==0.4.3
      - scikit-learn==1.0.2
      - scipy==1.2.2
      - six==1.17.0
      - soundfile==0.13.1
      - soxr==0.3.7
      - tensorboard==1.15.0
      - tensorflow==1.15.2
      - tensorflow-estimator==1.15.1
      - termcolor==2.3.0
      - threadpoolctl==3.1.0
      - tomli==2.0.1
      - torch==1.13.1
      - tqdm==4.67.1
      - typing-extensions==4.7.1
      - unidecode==1.0.22
      - urllib3==2.0.7
      - werkzeug==2.2.3
      - wrapt==1.16.0
      - zipp==3.15.0
prefix: /home/linux/miniconda3/envs/tacotron3.6_env`

.
.
.
.
**If this helps save your time, please give a thumbs up! Thanks!**
